docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -7,21 +7,19 @@ nav_order: 11
 
 # Azure Execution Provider (Preview)
 
-The Azure Execution Provider enables ONNX Runtime to invoke an remote Azure endpoint for inferenece, the endpoint must be deployed beforehand.
-To consume the endpoint, a model of same inputs and outputs must be loaded locally in the first place.
+The Azure Execution Provider enables ONNX Runtime to invoke a remote Azure endpoint for inference. The endpoint must be deployed beforehand.
+To consume the endpoint, a model of same inputs and outputs must be first loaded locally.
```

Comment:
Maybe with instead of of?

Reply:
That makes more sense - " a model with same inputs and outputs "

Reply:
Fixed